scripts/inc.installer.sh

```diff
@@ -2822,14 +2822,25 @@ bkup_header="#!/sbin/sh
 #
 # /system/addon.d/70-gapps.sh
 #
-. /tmp/backuptool.functions
 
-if [ -z \$backuptool_ab ]; then
-  SYS=\$S
-  TMP="/tmp"
+V1_FUNCS=/tmp/backuptool.functions
```

Comment:
this will get sed'ed to `/postinstall/tmp/backuptool.functions` by backuptool.sh
but you can trust $backuptool_ab without having to do this

Reply:
It actually won't because there sed is specifically looking for ". /tmp/backuptool.functions" but since this isn't a complex addon.d script which would require that I'll remove that commit.